src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        String randomColor = colorSupplier.getRandomColor();
+        int randomNumber = random.nextInt(101);
```

Comment:
to fix a magic number is to do something like this
```suggestion
    private static final int MAX_NUMBER = 101;

    public Ball getRandomBall() {
        String randomColor = colorSupplier.getRandomColor();
        int randomNumber = random.nextInt(MAX_NUMBER);
```